test/unit/CMakeLists.txt

```diff
@@ -30,6 +41,12 @@ function(create_unit_test)
   message(STATUS "Creating unit test ${UNIT_PREFIX}.test")
   add_executable(${UNIT_PREFIX}.test ${UNIT_SOURCES})
   target_link_libraries(${UNIT_PREFIX}.test ${UNIT_LIBRARIES})
+
+  # See LIBUNWIND_INCLUDE_DIR above for explanation.
+  if (ENABLE_BACKTRACE AND ENABLE_BUNDLED_LIBUNWIND)
+    add_dependencies(${UNIT_PREFIX}.test bundled-libunwind bundled-libunwind-platform)
+  endif()
```

Comment:
AFAIC, unit tests that use `fiber.h` depend either on `core` or `box` (which depends on `core`) library targets, so this is unnecessary:
https://github.com/tarantool/tarantool/blob/05002edf80d44a4a82da8be0d5e7bcda5349d47a/src/lib/core/CMakeLists.txt#L84-L89

Reply:
Your point looks valid, but I actually saw the fail without this dependency. I'll look around and return here with results.

Reply:
The reason is that `test/unit/uri.c` includes `fiber.h`, but doesn't use it. It has no dependency on `core` and it was the reason of the build fail I saw.

I updated the patchset by eliminating this `add_dependencies()` directive and removing the `fiber.h` include from `test/unit/uri.c`.

I visually verifed that all the unit tests (that fail without `libunwind.h`) have direct or indirect dependency on `core`. I tested the parallel build locally a couple of times and everything looks fine.

Thank you for spotting it!